packages/scaffolding-cli/templates/src/ssr/components/Header/__snapshots__/index.test.tsx.snap

```diff
@@ -39,7 +39,8 @@ exports[`renders Header snapshot 1`] = `
         style="margin: 0px auto;"
       >
         <a
-          class="css-15rvyqs"
+          class="makeStyles-styledLink-2"
+          href="undefined/"
```

Comment:
undefined?

Reply:
because the APP_BASE_PATH is undefined in the test. Fixed it now. Again good spotting.